DEVELOPMENT.md

```diff
@@ -0,0 +1,139 @@
+# Developing HADDOCK3
+
+This file provides information on how to setup a development environment for HADDOCK3.
+
+- [System requirements](#system-requirements)
+  - [Python3.9](#python39)
+  - [git](#git)
+  - [gcc](#gcc)
+  - [OpenMPI](#openmpi)
+- [Setting up the development environment](#setting-up-the-development-environment)
+  - [Clone the repository](#clone-the-repository)
+  - [Python environment](#python-environment)
+  - [Install Python dependencies](#install-python-dependencies)
+  - [Running tests](#running-tests)
+
+## System requirements
+
+- Python 3.9
+- CNS
+- git
+- gcc
+- OpenMPI
+
+### Installing system dependencies
+
+Below the instructions are provided for a Ubuntu system using `apt-get`. If you are using a different system, please refer to the respective package manager - `yum`, `dnf`, `pacman`, `homebrew`, `port`, etc.
+
+#### Python3.9
+
+**Conda is NOT recommended for development. Instead, install and compile Python 3.9 from source.**
+
+On a **Linux-based system** you can install Python 3.9 with the following commands:
+
+```bash
+wget https://www.python.org/ftp/python/3.9.6/Python-3.9.6.tgz
+tar -xf Python-3.9.6.tgz
+cd Python-3.9.6
+./configure --enable-optimizations
+sudo make altinstall -j 8
+```
+
+Then `python3.9` should be available on your system at `/usr/local/bin/python3.9`
+
+On **OSX**, use rather [brew](https://brew.sh) to install Python 3.9:
+
+```bash
+brew install python@3.9
+```
+
+Then `python3.9` should be available on your system at `/opt/homebrew/bin/python3.9`
+
+#### Git
+
+On a **Linux-based** system:
+
+```bash
+sudo apt-get install git
+```
+
+#### gcc
+
+On a **Linux-based** system:
+
+```bash
+sudo apt-get install gcc
+```
+
+#### OpenMPI
+
+On a **Linux-based** system:
+
+```bash
+sudo apt-get install openmpi-bin libopenmpi3 libopenmpi-dev
+```
+
+**Note** that this step is not required on OSX.
```

Comment:
Can someone with an OSX please confirm this? Sounds strange that openmi comes pre-installed @haddocking/haddock-developers

Reply:
Not functional.
I got an error running the `pip install mpi4py==3.1.6`
But `brew install mpi4py` was functional

Reply:
But indeed, no need of openmpi things